### PR TITLE
[FIX] fieldservice_route

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: fieldservice_route/models/fsm_order.py [hunk 1]
```diff
@@ -31,17 +49,19 @@ def _manage_fsm_route(self, vals):
...
-                })
+                fsm_route_obj.create(rec.prepare_dayroute_values(
+                    person_id, scheduled_date_start
+                ))
                 if old_route_id and not old_route_id.order_ids:
                     old_route_id.unlink()
 
     @api.model
     def create(self, vals):
+        if not vals.get('person_id'):
```
Comment (nikul-serpentcs) [inline fieldservice_route/models/fsm_order.py:61]:
```suggestion
        if not vals.get('person_id') and vals.get('location_id'):
```

**Review by nikul-serpentcs — APPROVED** [review]
nitpicking & Fix `Mr.Travis`
Otherwise Code Review LGTM :+1:

File: fieldservice_route/models/fsm_order.py (hunk 1 above)
Comment (max3903) [reply fieldservice_route/models/fsm_order.py:61]: @nikul-serpentcs location_id is a required field...

**Review by murtuzasaleh — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Review by osi-scampbell — APPROVED** [review]

**Pull request merged**